presentation/src/main/java/team/jsv/icec/ui/camera/CameraResultFragment.kt

```diff
@@ -1,16 +1,183 @@
 package team.jsv.icec.ui.camera
 
+import android.app.Activity
+import android.content.ContentValues
+import android.content.Intent
+import android.graphics.Bitmap
+import android.os.Build
+import android.os.Environment
+import android.provider.MediaStore
+import android.util.DisplayMetrics
+import android.view.ViewGroup
+import androidx.constraintlayout.widget.ConstraintSet
 import androidx.fragment.app.activityViewModels
 import team.jsv.icec.base.BaseFragment
+import team.jsv.icec.ui.main.MainActivity
+import team.jsv.icec.util.ConnenctState
+import team.jsv.icec.util.SettingViewUtil
 import team.jsv.presentation.R
 import team.jsv.presentation.databinding.FragmentCameraResultBinding
+import java.text.SimpleDateFormat
+import java.util.Locale
 
 class CameraResultFragment :
     BaseFragment<FragmentCameraResultBinding>(R.layout.fragment_camera_result) {
 
+    companion object {
+        const val JPEG_MIME_TYPE = "image/jpeg"
+        const val MEDIA_DIRECTORY_PATH = "/ICEC IMAGE"
+        const val EXTERNAL_STORAGE_DIRECTORY_PATH = "/Pictures/ICEC IMAGE"
+    }
+
     private val viewModel: CameraViewModel by activityViewModels()
 
+    private val deviceHeight get() = screenHeight(requireActivity())
+    private val deviceWidth get() = screenWidth(requireActivity())
+
+    override fun onResume() {
+        super.onResume()
+        initClickEvent()
+    }
+
+    private fun screenHeight(activity: Activity): Int {
```

Comment:
앞에 get을 붙여서 ScreenHeight를 가져온다는 표현을 함수에 명확하게 하는건 어떨까요?
```suggestion
    private fun getScreenHeight(activity: Activity): Int {
```

Reply:
반영했습니당